xlml/utils/gcluster.py

```
@@ -0,0 +1,451 @@
+        },
+    )
+    assert (
+        result.exit_code == 0
+    ), f"Cluster Toolkit submit failed with code {result.exit_code}"
+
+
+def _get_core_api_client(
+    project_id: str, region: str, cluster_name: str
+) -> k8s_client.CoreV1Api:
+  """Create a core API client for the given cluster."""
+  client = gke.get_authenticated_client(project_id, region, cluster_name)
```

Ruff (0.16.1)

[error] Undefined name `gke`

(F821)